System.IO.Ports/SerialData.cs

```diff
@@ -11,12 +11,12 @@ namespace System.IO.Ports
     public enum SerialData
     {
         /// <summary>
-        /// A character was received and placed in the input buffer.
+        /// A character was received in the input buffer.
```

Comment:
This was correct as it was. The behaviour is that a char was received by the UART and placed in the input buffer. If you want to be picky maybe you can replace with input _stream_ .

Reply:
Hopefully made more clear.